### ISSUE TITLE
SkyCoord

### ISSUE THREAD

**Issue by prajwel** [astropy/astropy#5726]
When I run the default example for SkyCoord on Astropy Doc page.
```python
>>> from astropy import units as u
>>> from astropy.coordinates import SkyCoord
>>> c = SkyCoord('00:42.5 +41:12', unit=(u.hourangle, u.deg))
```

I get the error 

> WARNING: Couldn't write lextab module u'angle_lextab'. [Errno 13] Permission denied: u'/usr/lib64/python2.7/site-packages/astropy/coordinates/angle_lextab.py'
WARNING: Couldn't create u'angle_parsetab'. [Errno 13] Permission denied: u'/usr/lib64/python2.7/site-packages/astropy/coordinates/angle_parsetab.py'

This was absent when I was using 1.2.1. Recently upgraded to 1.3. Is this a bug?

**Comment by astrofrog** [comment]
@prajwel - how did you install Astropy? And what does:

```
ls -l /usr/lib64/python2.7/site-packages/astropy/coordinates
```

give?

**Comment by saimn** [comment]
Looks like the same as #5581 (cc @bsipocz ). For some reason this file is recreated, which causes an error when you don't have the right to write in the `site-packages` folder.

**Comment by prajwel** [comment]
@astrofrog I used to `pip` to upgrade Astropy. 

Here's the output. 

```bash
[prajwel@poc ~]$ ls -l /usr/lib64/python2.7/site-packages/astropy/coordinates
total 852
-rw-r--r--. 1 root root  3441 Jan 21 12:16 angle_lextab.py
-rw-r--r--. 1 root root  3814 Jan 21 12:16 angle_lextab.pyc
-rw-r--r--. 1 root root  4813 Jan 21 12:16 angle_parsetab.py
-rw-r--r--. 1 root root  4947 Jan 21 12:16 angle_parsetab.pyc
-rw-r--r--. 1 root root 27038 Jan 21 12:16 angles.py
-rw-r--r--. 1 root root 27647 Jan 21 12:16 angles.pyc
-rw-r--r--. 1 root root 20271 Jan 21 12:16 angle_utilities.py
-rw-r--r--. 1 root root 23630 Jan 21 12:16 angle_utilities.pyc
-rw-r--r--. 1 root root 52476 Jan 21 12:16 baseframe.py
-rw-r--r--. 1 root root 47512 Jan 21 12:16 baseframe.pyc
drwxr-xr-x. 2 root root  4096 Jan 21 12:16 builtin_frames
-rw-r--r--. 1 root root  5043 Jan 21 12:16 calculation.py
-rw-r--r--. 1 root root  5419 Jan 21 12:16 calculation.pyc
drwxr-xr-x. 2 root root    90 Jan 21 12:16 data
-rw-r--r--. 1 root root  7323 Jan 21 12:16 distances.py
-rw-r--r--. 1 root root  7218 Jan 21 12:16 distances.pyc
-rw-r--r--. 1 root root 14062 Jan 21 12:16 earth_orientation.py
-rw-r--r--. 1 root root 14530 Jan 21 12:16 earth_orientation.pyc
-rw-r--r--. 1 root root 23750 Jan 21 12:16 earth.py
-rw-r--r--. 1 root root 24351 Jan 21 12:16 earth.pyc
-rw-r--r--. 1 root root  4696 Jan 21 12:16 errors.py
-rw-r--r--. 1 root root  7779 Jan 21 12:16 errors.pyc
-rw-r--r--. 1 root root  9915 Jan 21 12:16 funcs.py
-rw-r--r--. 1 root root 10066 Jan 21 12:16 funcs.pyc
-rw-r--r--. 1 root root  1207 Jan 21 12:16 __init__.py
-rw-r--r--. 1 root root  1374 Jan 21 12:16 __init__.pyc
-rw-r--r--. 1 root root 20005 Jan 21 12:16 matching.py
-rw-r--r--. 1 root root 17034 Jan 21 12:16 matching.pyc
-rw-r--r--. 1 root root  4201 Jan 21 12:16 matrix_utilities.py
-rw-r--r--. 1 root root  5097 Jan 21 12:16 matrix_utilities.pyc
-rw-r--r--. 1 root root  5875 Jan 21 12:16 name_resolve.py
-rw-r--r--. 1 root root  6161 Jan 21 12:16 name_resolve.pyc
-rw-r--r--. 1 root root  8215 Jan 21 12:16 orbital_elements.py
-rw-r--r--. 1 root root  9444 Jan 21 12:16 orbital_elements.pyc
-rw-r--r--. 1 root root 45269 Jan 21 12:16 representation.py
-rw-r--r--. 1 root root 49707 Jan 21 12:16 representation.pyc
-rw-r--r--. 1 root root   260 Jan 21 12:16 setup_package.py
-rw-r--r--. 1 root root   633 Jan 21 12:16 setup_package.pyc
-rw-r--r--. 1 root root  4698 Jan 21 12:16 sites.py
-rw-r--r--. 1 root root  5764 Jan 21 12:16 sites.pyc
-rw-r--r--. 1 root root 69657 Jan 21 12:16 sky_coordinate.py
-rw-r--r--. 1 root root 57919 Jan 21 12:16 sky_coordinate.pyc
-rw-r--r--. 1 root root 18963 Jan 21 12:16 solar_system.py
-rw-r--r--. 1 root root 17299 Jan 21 12:16 solar_system.pyc
drwxr-xr-x. 3 root root  4096 Jan 21 12:16 tests
-rw-r--r--. 1 root root 33474 Jan 21 12:16 transformations.py
-rw-r--r--. 1 root root 31063 Jan 21 12:16 transformations.pyc
```

**Comment by mhvk** [comment]
@prajwel - this is fixed in #5728, which should appear in the next bug-fix release. Thanks for reporting!!
